app/Comment.php

```diff
@@ -3,13 +3,15 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Http\Request;
 use App\Post;
+use App\User;
 
 class Comment extends Model
 {
     //attributes id, description, post_id, created_at, updated_at
-    protected $fillable = ['description', 'post_id'];
-    
+    protected $fillable = ['description', 'post_id', 'user_id'];
+
```

Comment:
Separe attributes in different lines and finish the last one with a comma. It makes versioning easier.

$fillable = [
 user_id,
 description,
 post_id,
];

Reply:
Done. About the forum layout, we are over time, so I think it is no priority now. We can leave it like that.